spec/spec_helper.rb

```diff
@@ -10,6 +11,15 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+VCR.configure do |c|
+  c.cassette_library_dir = 'fixtures/vcr_cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.